packages/cursorless-engine/src/actions/BringMoveSwap.ts

```diff
@@ -133,7 +116,9 @@ class BringMoveSwap implements Action {
         usedSources.push(source);
         if (this.type !== "move") {
           results.push({
-            edit: source.constructChangeEdit(destination.contentText),
+            edit: source
+              .toDestination("to")
+              .constructChangeEdit(destination.target.contentText),
             editor: source.editor,
             originalTarget: source,
             isSource: true,
```

Comment:
pretty sure this case is only for swap, which is now handled separately, so you should be able to remove this `if` statement altogether.  Then you can also remove the filtering here https://github.com/cursorless-dev/cursorless/blob/6c95c6d7da735a3f12e0976a17326e86a44109fa/packages/cursorless-engine/src/actions/BringMoveSwap.ts#L154-L157

Reply:
Oh actually this will break our handling of source marks right?  That all feels pretty hacky now.  I'm ok with leaving this for now and I can do a quick follow-up in #1608 to clean things up

Reply:
Correct